src/Widgets/LanguageSwitchWidget/LanguageSwitchWidgetComponent.tsx

```diff
@@ -0,0 +1,16 @@
+import { provideComponent } from 'scrivito'
+import { LanguageSwitchWidget } from './LanguageSwitchWidgetClass'
+import { LanguageSwitch } from '../TopNavigationWidget/SubComponents/LanguageSwitch'
+import { alignmentClassName } from '../../utils/alignmentClassName'
+
+provideComponent(LanguageSwitchWidget, ({ widget }) => {
+  const classNames = ['navbar-single']
+  const alignment = alignmentClassName(widget.get('alignment'))
+  if (alignment) classNames.push(alignment)
+
+  return (
+    <div className={classNames.join(' ')}>
+      <LanguageSwitch />
+    </div>
```

Comment:
* [ ] FYI Maybe it would be nice to sync the dropdown position with the alignment. https://mui.com/material-ui/react-popper/?srsltid=AfmBOopIT0g7xYfZG0Zjg4Iz7eKspZFF0j3eWMyhNadhe5nOSIgBshKq#positioned-popper

Reply:
Nice suggestion.